src/new-components/tooltip/tooltip.stories.tsx

```diff
@@ -127,3 +128,42 @@ TooltipRichContentStory.argTypes = {
         control: { type: 'select', options: positions },
     },
 }
+
+// This story sets a new z-index for the tooltip, but it will leak into other stories
+// when viewed from the Docs page, since all stories live in a single iframe.
+// Only in the Canvas page will stories be isolated from each other
+export function TooltipZIndexStory() {
+    return (
+        <>
+            <PortalToHead>
+                <style>
+                    {`:root {
+                        --reactist-tooltip-zindex: 10000;
+                    }`}
+                </style>
+            </PortalToHead>
+            <Stack space="medium">
+                <Text>
+                    The tooltip‘s z-index can be customized using the
+                    <em>--reactist-tooltip-zindex</em> custom property:
+                </Text>
+                <Box
+                    background="aside"
+                    padding="large"
+                    position="relative"
+                    style={{ zIndex: 2000 }}
+                >
+                    <Tooltip content="I have a custom z-index">
+                        <Button variant="secondary">
+                            Hover or focus to see the tooltip in action
+                        </Button>
+                    </Tooltip>
+                </Box>
+            </Stack>
+        </>
+    )
+}
+
+function PortalToHead({ children }: React.PropsWithChildren<unknown>) {
+    return createPortal(children, document.head)
+}
```

Comment:
nit: I do not think we need to create a story for this.

Reply:
I'm inclined to keep it, not just to provide this tidbit but also as a way to test the bug/fix in storybook. Let me know if you feel more strongly here 👍